src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  display: flex;
```

Comment:
Avoid using the `display` property in the `body` selector. Create a div container instead.